.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseVersion:
+        description: "Version to use for the release."
+        required: true
+        default: "X.Y.Z"
+      developmentVersion:
+        description: "Version to use for further development"
+        required: true
+        default: "X.Y.Z-RIE"
+jobs:
+  Release:
+    environment: Release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Configure Git User # may be removed after https://github.com/actions/checkout/issues/13 is resolved
+        run: |
+          git config user.email "${{ github.actor }}@users.noreply.github.com"
+          git config user.name "${{ github.actor }}"
```

Comment:
Did you try it without this? I'm not sure if this is needed for our use case here.